api/services/SiteCreator.js

```diff
@@ -82,6 +82,7 @@ const createSiteFromExistingRepo = ({ siteParams, user }) => {
 const createSiteFromTemplate = ({ siteParams, user, template }) => {
   let site = Site.build(siteParams)
   site.engine = "jekyll"
+  site.defaultBranch = templateForTemplateName(template).branch
```

Comment:
ok cool. I didn't find the loading of the template on the back-end. We're good to go.